app/i18n/ru/conf.php

```diff
@@ -57,7 +57,7 @@
 			'_' => 'Тема',
 			'deprecated' => array(
 				'_' => 'Deprecated',	// TODO
```

Comment:
One remaining one here, if you can 🤓

Reply:
I thought it would be more suitable in english, so i kept it. It's a common term